app/src/components/HomePage/DownloadAndShareModals/PublishedView.tsx

```diff
@@ -47,7 +47,7 @@ export function PublishedView({
       locale: getLocale(lng),
     });
 
-  const URL = `${window.location.protocol}//${window.location.host}${window.location.port ? `:${window.location.port}` : ""}/${lng}/public/${inventoryId}`;
+  const URL = `${window.location.protocol}//${window.location.host}/${lng}/public/${inventoryId}`;
```

Comment:
Why was the port removed here?

Reply:
because it was causing the url to be localhost:3000:3000. I tested with other ports and it still works.